spec/mailers/avis_mailer_spec.rb

```diff
@@ -16,5 +16,11 @@
     context 'when the recipient is not already registered' do
       it { expect(subject.body).to include(sign_up_expert_avis_url(avis.dossier.procedure.id, avis.id, avis.expert.email)) }
     end
+
+    context 'when the dossier has been hidden' do
```

Comment:
Du coup je propose :

```suggestion
    context 'when the dossier has been deleted before the avis was sent' do
```